admin.py

```diff
@@ -460,8 +460,10 @@ def post(self, path, feature_id=None):
       if feature is None:
         return self.redirect(self.request.path)
 
+      # TODO(jrobbins): implement undelete UI.  For now, use cloud console.
       if 'delete' in path:
-        feature.delete()
+        feature.deleted = True
```

Comment:
Could we make this into a more generic status attribute?  That would open the door for some future tool changes (e.g., proposals, drafts, duping, etc.).

Reply:
I came back to implement this, but then I remembered a good reason to use a separate boolean for deleted even if we use an enum for some other future state values:  When you undelete, you want the object to go back to the state that it was in before it was deleted.  Using a state enum would overwrite the previous state enum value when deleting, so the information is no longer there at the time of undeleting.

Reply:
Got it.  Taking the example of duping, transitioning the a deleted state would be meaningfully lossy and we'd want to restore the original pre-deleted state.  To preserve that we'd need to introduce some type of historic field and synchronize, which is more complex than just having a separate state + a boolean for deleted.

Reply:
In that case, let's call adding a formal document/ feature state as a separate, independent workitem.

Reply:
OK, I will keep the deleted boolean in this CL and add a TODO to add an entry_state enum in the future.